### ISSUE TITLE
Multiple refunds not showing

### ISSUE THREAD

**Issue by zenweasel** [reactioncommerce/reaction#2285]
Similar but not the same as #2215 

### Expected behavior
When I enter multiple refunds, they should be displayed

### Actual Behavior
No refunds are listed and only the first one is reflected in the "Adjusted Total"
![cart_completed](https://cloud.githubusercontent.com/assets/72819/26132130/b1a57724-3ad0-11e7-9a80-d4e6f63ff126.png)

### Steps to Reproduce the Behavior
1. Place and Approve/Capture an order using Paypal Express
1. Issue several refunds
1. Observe they are not reflected

**Note** The refunds did come up **once**, and then threw an error of `Cannot call split on undefined` in the refund/create method. So I am not sure what exactly is going on there.

### Versions
```
Node: 7.10.0
NPM: 4.2.0
Meteor Node: 4.8.2
Meteor NPM: 4.5.0
Reaction CLI: 0.8.2
Reaction: 1.2.0
Reaction branch: release-1.2.0
Docker: 17.03.1-ce
```

**Comment by zenweasel** [comment]
Actually the same thing is happening with Stripe. The method is being called (listRefunds) but the data is not floating up to the UI

**Comment by michael-alade** [comment]
Found the solution for this ticket by updating the `canMakeAdjusments` helper for `shippingInvoice.js`. Preparing a PR for this.

**Comment by zenweasel** [comment]
Closed via #2317
